packages/semver/src/executors/version/version.ts

```diff
@@ -137,14 +141,20 @@ export function _runStandardVersion({
   bumpFiles: string[];
   skipChangelog: boolean;
 } & CommonVersionOptions) {
-  return standardVersion({
+  const changeLogPath = getChangelogPath(projectRoot);
+
+  const currentChangeLog = await firstValueFrom(
+    readFileIfExists(changeLogPath, changelogHeader)
+  );
```

Comment:
Having to work with observables when doing this kinda of synchronous logic is a bit cumbersome, so getting out promises makes things easier to follow